spec/abilities/ability_spec.rb

```diff
@@ -98,9 +98,11 @@
     it { is_expected.not_to be_able_to(:update, ContentBlock) }
     it { is_expected.not_to be_able_to(:create, AdminSet) }
     it { is_expected.to be_able_to(:read, ContentBlock) }
-    it { is_expected.to be_able_to(:read, GenericWork) }
-    it { is_expected.to be_able_to(:stats, GenericWork) }
-    it { is_expected.to be_able_to(:citation, GenericWork) }
+    unless I18n.t('hyrax.product_name') == 'Koppie'
```

Comment:
```suggestion
    unless Hyrax.config.disable_wings
```
I'd rather stick to checking disable_wings for this.